packages/@tinacms/graphql/src/resolver/media-utils.ts

```diff
@@ -23,19 +24,28 @@ import { GraphQLConfig } from '../types'
 
 export const resolveMediaCloudToRelative = (
   value: string,
-  config: GraphQLConfig = { useRelativeMedia: true }
+  config: GraphQLConfig = { useRelativeMedia: true },
+  schema: TinaCloudSchemaEnriched
 ) => {
   if (config && value) {
     if (config.useRelativeMedia === true) {
       return value
-    } else {
+    }
+
+    if (isValidTinaSchema(schema) === true) {
       const assetsURL = `https://${config.assetsHost}/${config.clientId}`
       if (typeof value === 'string' && value.includes(assetsURL)) {
-        return value.replace(assetsURL, '')
-      } else {
-        return value
+        const cleanSyncFolder = cleanUpSlashes(
```

Comment:
I also made a very similar  function. We should probable use the same one once everything is merged.